HW0/P4/P4.py

```diff
@@ -29,46 +29,94 @@
     #
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
+
+    s_true = np.loadtxt("P4_trajectory.txt", delimiter = ',')
+
+    x_coords, y_coords, z_coords = s_true[:,0], s_true[:,1], s_true[:, 2]
+
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    ax.plot(x_coords, y_coords, z_coords, '--b', label='True trajectory')
+
+    #plt.show()
 
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
+    s_measure = np.loadtxt("P4_measurements.txt", delimiter = ',')
+
+    x_measure, y_measure, z_measure = s_measure[:,0]/rx, s_measure[:,1]/ry, s_measure[:, 2]/rz
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    ax.plot(x_measure, y_measure, z_measure, '.g', label='Observed trajectory')
+
+    #plt.show()
 
     #####################
     # Part 3:
     # Use the initial conditions and propagation matrix for prediction
     #####################
 
-    # A = ?
-    # a = ?
-    # s = ?
+    A = np.array([[1,0,0,dt,0,0],[0,1,0,0,dt,0],[0,0,1,0,0,dt],
+[0,0,0,1-c*dt,0,0],[0,0,0,0,1-c*dt,0],[0,0,0,0,0,1-c*dt]])
```

Comment:
If you are using Sublime, it may be helpful to install the package 'autopep8' to automatize the corrections related to the rules from pep8. It helps have a cleaner and more spread code.